ui/src/context/Application.jsx

```diff
@@ -34,147 +30,106 @@ export { walletP };
 
 export const ApplicationContext = createContext();
 
-export function useApplicationContext() {
-  return useContext(ApplicationContext);
-}
+export const useApplicationContext = () => useContext(ApplicationContext);
 
-const watchPool = async (dispatch, ammAPI, brand) => {
-  const updatePoolState = value => {
-    console.log('updating pool state', brand, value);
-    dispatch(setPoolState({ brand, value }));
-  };
-  const poolUpdater = async () => {
-    const poolSubscription = await E(ammAPI).getPoolMetrics(brand);
-    const poolNotifier = makeNotifierFromAsyncIterable(poolSubscription);
-    for await (const value of iterateNotifier(poolNotifier)) {
-      updatePoolState(value);
+const watchOffers = dispatch => {
+  async function offersUpdater() {
+    const offerNotifier = E(walletP).getOffersNotifier();
+    for await (const offers of iterateNotifier(offerNotifier)) {
+      console.log('======== OFFERS', offers);
+      dispatch(updateOffers(offers));
     }
-  };
-  poolUpdater().catch(err =>
-    console.error('Brand watcher exception', brand, err),
-  );
+  }
+  offersUpdater().catch(err => console.error('Offers watcher exception', err));
+};
+
+const watchPoolMetrics = async (dispatch, brand, i) => {
+  const f = E(walletP).makeFollower(`:published.amm.pool${i}.metrics`);
+  for await (const { value: state } of iterateLatest(f)) {
+    dispatch(setPoolState({ brand, value: state }));
```

Comment:
`state` is almost a keyword in JSX. it's also funny to "setPoolState" to something that has a "value" which is "state".

Consider leaving this `value`
```suggestion
  for await (const { value } of iterateLatest(f)) {
    dispatch(setPoolState({ brand, value }));
```

Reply:
Done